### ISSUE TITLE
Ho to hide interactive figure button?

### ISSUE THREAD

**Issue by kolibril13** [executablebooks/mystmd#461]
### What version of `mystmd` are you using?

latest

### How did you install myst?

npm

### What operating system are you using?

Mac

### Which area is this feature request for?

Other

### Describe the Bug

After updating the GitHub actions in https://octoframes.github.io/myst-online-media-testing/
I've noted these new buttons:
<img width="872" alt="image" src="https://github.com/executablebooks/mystmd/assets/44469195/96f9255a-5d04-4cdf-973d-4b97aaf51843">

I'm really looking forward to having thebe in myst, but in this media example it's not required, it's actually more confusing than helpful to have this button that has no effect in the media example.

Therefore, the question: is there currently a way to disable this feature on Github Pages? 

I've tried

https://github.com/Octoframes/myst-online-media-testing/blob/3a4131e9f708d19a797be4cb68855b2e3d1d37b2/myst.yml#L9

but that had no effect.

Also, is there maybe a way to pin the Myst version in the GitHub action?
That way, the GitHub page would always stay reproducible.

**Comment by rowanc1** [comment]
Thanks for the report. We will get a way to turn those off shortly. For showing the source of the notebook, is that something you think we should be showing?

For the reproducible builds, I will open another issue. We don't have versioning in the templates at this stage, which is where this is coming from. I think this is also somewhat related to #453.

**Comment by kolibril13** [comment]
> We will get a way to turn those off shortly

Amazing!

> For showing the source of the notebook, is that something you think we should be showing?

I would prefer to not show them.
When I read or write tutorials, I actually like it to stay in a focused section without many links that make me leave the current context. 
"Source of the notebook" links would lead to a different side and away from the current focused section.
What makes me like Myst so much is that reference previews can give me context within the current focus 🧡

Another thought: Scientific python code e.g. for creating a plot often is quickly written one-time use throw away code (from my experience that code often comes from some internet resources with some small adaptions). 
so it should not be highlighted too much in an article that actually wants to explain a phenomenon.
It's nice when the notebook that generated the plot is accessible as supplementary material, but I think that directly linking it every time would put disproportionally emphasize the tool that creates the content than the content itself. 

> For the reproducible builds, I will open another issue

nice!

**Comment by rowanc1** [comment]
We have made this a lot more subtle. Curious what you think of the new updates @kolibril13.

![image](https://github.com/executablebooks/mystmd/assets/913249/1e42e128-3ca7-4ded-993a-3eeda353aeb0)

**Comment by kolibril13** [comment]
thanks for the quick update!

References and links can be powerful in the right context, but I think it should always be a conscious decision of the author to link the reader to the code that generated the figures, and not be the default behavior.

Also I think that indicating the origin of the figure is mostly a source of distraction, and adds unnecessary clutter to the page.
The layout on the left is more appealing than the right one in my opinion.
<img width="693" alt="image" src="https://github.com/executablebooks/mystmd/assets/44469195/0cb0dc5d-0465-448e-a136-51c891c2f63f">

**Comment by stevejpurves** [comment]
Yes, layout wise because of the difficulty in placing a link just under the end of the content, left is better. But perhaps imcluding a link on figures should be an option the theme, so that (a) authors can opt in and (b) link/frame doesn't have to try to be so subtle. Also of course authors can add a link themselves in the caption should they want to.

It's also a pity this clashes with the caption a bit too, and isn't part of it.

**Comment by kolibril13** [comment]
> (a) authors can opt in and (b) link/frame doesn't have to try to be so subtle

having an opt-in version and a default to not show the link sounds good to me!

do you think that the opt-in should be figure wise or document wise? (or maybe even both?)

for embedded dataframes and tweets, it also looks quite off (screenshot from here: https://octoframes.github.io/myst-online-media-testing/hello#fig-dataframe):
<img width="343" alt="image" src="https://github.com/executablebooks/mystmd/assets/44469195/9d49df68-5a32-465a-be47-87f020d2041c">
Another idea: Changing `Source: Jupyter Notebook` to `Source: Jupyter` and shifting it to the left.

<img width="316" alt="image" src="https://github.com/executablebooks/mystmd/assets/44469195/71d7891c-6174-47c2-bad3-80143d8a2238">

I am also not too convinced with that layout.

Another subtle version: Just having the Jupyter icon at the bottom right of the asset.

<img width="462" alt="image" src="https://github.com/executablebooks/mystmd/assets/44469195/d537c0a6-8c40-4021-a077-9979095d5cd5">
